common/taskana-common-test/src/main/java/pro/taskana/common/test/OracleSchemaHelper.java

```diff
@@ -0,0 +1,76 @@
+package pro.taskana.common.test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+import javax.sql.DataSource;
+
+import pro.taskana.common.api.exceptions.SystemException;
+
+public final class OracleSchemaHelper {
+
+  private static final String DEFAULT_PASSWORD = "testPassword";
+
+  private OracleSchemaHelper() {
+    // hide implicitpublic one
+  }
+
+  public static void initOracleSchema(DataSource dataSource, String schemaName)
+      throws SystemException {
+    try (Connection connection = dataSource.getConnection()) {
+      Connection conn =
+          DriverManager.getConnection(
+              connection.getMetaData().getURL(), "SYSTEM", DEFAULT_PASSWORD);
+      try (Statement stmt = conn.createStatement()) {
+        stmt.execute("GRANT ALL PRIVILEGES TO TEST_USER");
+
+        if ("TASKANA".equals(schemaName)) {
+          dropExistingSchema(schemaName, stmt);
+        }
```

Comment:
please do this always. This is the "legacy" version of this class

Reply:
Done